aether/sdk/conf/settings.py

```diff
@@ -307,16 +307,17 @@ def get_required(name):
     CACHEOPS_ENABLED = not TESTING  # disable on tests
 
     # Cache Redis Sessions using django-redis-sessions
-    SESSION_REDIS = {
-        'host': REDIS_HOST,
-        'port': REDIS_PORT,
-        'db': REDIS_DB_CACHEOPS,
-        'password': REDIS_PASSWORD,
-        'prefix': 'session',
-        'socket_timeout': 3,
-        'retry_on_timeout': True
-    }
-    SESSION_ENGINE = 'redis_sessions.session'
+    if bool(os.environ.get('REDIS_SESSION_CACHE', True)):
```

Comment:
I would also include the `TESTING` condition `if not TESTING and bool(os.environ.get('REDIS_SESSION_CACHE', True)):`